aws/table_aws_ssm_document.go

```diff
@@ -0,0 +1,262 @@
+				Transform:   transform.FromField("Tags"),
+			},
+
+			// Standard columns
+			{
+				Name:        "tags",
+				Description: resourceInterfaceDescription("tags"),
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.From(ssmDocumentTagListToTurbotTags),
+			},
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Name"),
+			},
+			{
+				Name:        "akas",
+				Description: resourceInterfaceDescription("akas"),
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getAwsSSMDocumentAkas,
+				Transform:   transform.FromValue(),
+			},
+		}),
+	}
+}
+
+//// LIST FUNCTION
+
+func listAwsSSMDocuments(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	plugin.Logger(ctx).Trace("listAwsSSMDocuments", "AWS_REGION", region)
+
+	// Create session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// List call
+	err = svc.ListDocumentsPages(
+		&ssm.ListDocumentsInput{},
+		func(page *ssm.ListDocumentsOutput, isLast bool) bool {
+			for _, documentIdentifier := range page.DocumentIdentifiers {
+				d.StreamListItem(ctx, &ssm.DocumentDescription{
+					Name:            documentIdentifier.Name,
+					TargetType:      documentIdentifier.TargetType,
+					DocumentVersion: documentIdentifier.DocumentVersion,
+					Owner:           documentIdentifier.Owner,
+					VersionName:     documentIdentifier.VersionName,
+					DocumentFormat:  documentIdentifier.DocumentFormat,
+					PlatformTypes:   documentIdentifier.PlatformTypes,
+					ReviewStatus:    documentIdentifier.ReviewStatus,
+					Tags:            documentIdentifier.Tags,
+					DocumentType:    documentIdentifier.DocumentType,
+				})
+
+			}
+			return !isLast
+		},
+	)
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getAwsSSMDocument(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	logger.Trace("getAwsSSMDocument")
+
+	// TODO put me in helper function
+	var region string
+	matrixRegion := plugin.GetMatrixItem(ctx)[matrixKeyRegion]
+	if matrixRegion != nil {
+		region = matrixRegion.(string)
+	}
+	var name string
+	if h.Item != nil {
+		name = *h.Item.(*ssm.DocumentDescription).Name
+	} else {
+		name = d.KeyColumnQuals["name"].GetStringValue()
+	}
+
+	// Create Session
+	svc, err := SsmService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	// Build the params
+	params := &ssm.DescribeDocumentInput{
+		Name: &name,
+	}
+
+	// Get call
+	data, err := svc.DescribeDocument(params)
+	if err != nil {
+		logger.Debug("getAwsSSMDocument", "ERROR", err)
+		return nil, err
+	}
+
+	return data.Document, nil
```

Comment:
Can we return only `data` and handle the same in columns (i.e. by using `transform.FromField`)
```suggestion
	return data, nil
```